src/librustc/session/search_paths.rs

```diff
@@ -8,6 +8,7 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+use std::convert::From;
```

Comment:
It's in the prelude, yes; I probably added a few redundant imports mechanically.